opencost/src/settings.tsx

```diff
@@ -0,0 +1,85 @@
+import { NameValueTable } from '@kinvolk/headlamp-plugin/lib/components/common';
+import Box from '@mui/material/Box';
+import Checkbox from '@mui/material/Checkbox';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import TextField from '@mui/material/TextField';
+
+export function Settings(props) {
+  const { data, onDataChange } = props;
+
+  const settingsRows = [
+    {
+      name: 'Enable in list view',
+      value: (
+        <Checkbox
+          checked={data?.isEnabledInListView}
+          onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
+            onDataChange({ ...data, isEnabledInListView: event.target.checked })
+          }
+        />
+      ),
+    },
+    {
+      name: 'Namespace',
+      value: (
+        <TextField
+          fullWidth
+          helperText="Enter the namespace of Opencost service"
+          defaultValue={data?.namespace ? data.namespace : ''}
+          onChange={e => onDataChange({ ...data, namespace: e.target.value })}
+          variant="standard"
+        />
+      ),
+    },
+    {
+      name: 'Service URL',
+      value: (
+        <TextField
+          fullWidth
+          helperText="Enter the service URL of Opencost web UI service"
+          defaultValue={data?.serviceURL ? data.serviceURL : ''}
+          onChange={e => onDataChange({ ...data, serviceURL: e.target.value })}
+          variant="standard"
+        />
+      ),
+    },
+    {
+      name: 'Default timespan',
+      value: (
+        <>
+          <Select
+            value={data?.defaultTimespan}
+            onChange={e => onDataChange({ ...data, defaultTimespan: e.target.value })}
+          >
+            <MenuItem value={'today'}>Today</MenuItem>
+            <MenuItem value={'yesterday'}>Yesterday</MenuItem>
+            <MenuItem value={'24h'}>24 hours</MenuItem>
+            <MenuItem value={'48h'}>48 hours</MenuItem>
+            <MenuItem value={'week'}>Week</MenuItem>
+            <MenuItem value={'lastweek'}>Last week</MenuItem>
+            <MenuItem value={'7d'}>7 days</MenuItem>
+            <MenuItem value={'14d'}>14 days</MenuItem>
+          </Select>
+        </>
+      ),
+    },
+    {
+      name: 'Display currency',
```

Comment:
It'd be great if open cost also provided the currency name/symbol so we could use it by default.

Reply:
Nope, opencost doesn't provide the symbol it just prepends the symbol the user selects in the UI.